ui/src/pages/project/template.js

```diff
@@ -537,7 +537,10 @@ const Project = ({ location }) => {
   if (project) {
     tweet = `We’d like to claim the ${project.name} project on @EverestRegistry. Please transfer ownership to ${account} 🙌`
   }
-
+  
+  if (project.website && !project.website.includes('http'))
```

Comment:
I'd suggest to use `project.website.startsWith` instead.

Reply:
Done!